DigitalOffice/Services/TimeManagementService/Database/Entities/WorkTime.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace TimeManagementService.Database.Entities
+{
+    public class WorkTime
+    {
+        [Key]
+        public Guid Id { get; set; }
+        public Guid WorkerUserId { get; set; }
+        public DateTime Date { get; set; }
```

Comment:
Why you do not have a connecting entity with UserService? WorkerUserId should be binding with UserService.